plugin/builtin/suffixtransformer/SuffixTransformer.go

```diff
@@ -65,20 +62,17 @@ func (p *plugin) Transform(m resmap.ResMap) error {
 				continue
 			}
 			// TODO: move this test into the filter.
-			if smellsLikeANameChange(&fs) {
+			if fs.Path == "metadata/name" {
```

Comment:
same suggestion here to put "metadata/name" as a constant at the top of this file